app/views/cp_mailer/contract_email.text.erb

```diff
@@ -1,5 +1,5 @@
 
-Congratulations, <%= @offer[:applicant][:first_name] %> <%= @offer[:applicant][:last_name] %>
+Dear <%= @offer[:applicant][:first_name] %> <%= @offer[:applicant][:last_name] %>,
 ====================================
 
 You have successfully been choosen to TA <%= @offer[:position] %>
```

Comment:
We have offered you a position as a TA for <%= @offer[:position] %>, but we have not yet received notice of whether you plan to accept or reject the offer.